### ISSUE TITLE
Strange problem

### ISSUE THREAD

**Issue by drywash** [mnot/redbot#199]
I recently moved a domain from one server to another.   As soon as I moved the same domain with the same exactly web pages / html / scripts etc..,  redbot has begun to fail to show any responses on this domain.   Please advise why this is happening and what I can do to fix it.

http://www.suspenders-online.com/

Thank you,

David
(561) 347-7930

**Comment by mnot** [comment]
I can't connect to that server from home or from the datacentre where REDbot is hosted. Both are in Australia; is your provider blocking .au IPs, or using a blocklist of some sort, perhaps?

**Comment by drywash** [comment]
Dear Redbot,

I am running cpanel with a firewall blocking numerous countries.   I removed .AU but the problem continued.   .AU is removed but you can still test 

http://www.suspenders-online.com/

and see that it does not respond.   What other countries must I remove from the firewall to get this to work properly?     Or, can you give me a list / range of IP addresses to whitelist?

Thank you,

David
(561) 347-7930

**Comment by mnot** [comment]
Yep, still seems to be blocked from my home connection. I can't advise about someone else's product; best contact support for your Web host.

**Comment by drywash** [comment]
I am asking about redbot (not my server).   I am asking you what countries redbot servers are located in and whether you can also provide me a list of IP or IP range which I can whitelist for redbot.

Thank you

**Comment by mnot** [comment]
As I said above, it's in Australia. It has a single IP address, but there's no guarantee that it won't change.

**Comment by drywash** [comment]
Please provide me the current IP address(s) for any and all Redbot servers.   Thank you.

**Comment by plambert** [comment]
The redbot.org service is provided for free to anyone who does not block it.  The problem seems to be with your site blocking the IP for redbot.org and should be addressed there.

You can look up the current IP address for a server yourself far more easily than asking someone else to do it for you. Your ISP may be able to explain how to do that, if you’re not sure how.

Regardless, it is definitely the case that the redbot.org IP address could change at any time, and if you are blocking the new address, then you’ll have the same problem again.  This is your responsibility to resolve, since it is your choice to block IP addresses.

In my experience, trying to block countries by IP is ineffective and causes a lot of problems while solving none.

Good luck with your endeavors.

**Comment by drywash** [comment]
1.
The reason I asked is that I have already whitelisted 119.9.43.241.   I got this IP by doing an nslookup on redbot.org.    My concern is, that when I go to:  http://redbot.org  and type in our domain,  http://www.suspenders-online.com/  ,   it is my opinion that it may be executing a script from a different server / IP address than the one we have whitelisted.   This is why we continue to get no results when I test this domain through your web site.   This is why I am asking you for the IP(s) which do redbot's communications out to a remote site.   I would therefore greatly appreciate your letting me know what IP(s) I can whitelist to get redbot to work (for testing purposes).   I completely realize that those IP addresses may change.   

2.
Also, what sort of actions does redbot do when testing a remote site?    simply an httpd get of the headers or some other request on port 80?    Or does it do any other sort of communications on different ports than 80?   If so, what ports does it attempt to communicate out on?

Thank you,

David
